app/jobs/socure_docv_results_job.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class SocureDocvResultsJob < ApplicationJob
+  queue_as :default
+
+  attr_reader :document_capture_session_uuid, :service_provider_issuer, :user_uuid
+
+  # @param [String] document_capture_session_uuid
+  def perform(document_capture_session_uuid:)
+    @document_capture_session_uuid = document_capture_session_uuid
+
+    dcs = DocumentCaptureSession.find_by(uuid: document_capture_session_uuid)
+    raise "DocumentCaptureSession not found: #{document_capture_session_uuid}" if !dcs
+
+    # analytics = create_analytics(
+    #   user: dcs.user,
+    #   service_provider_issuer: dcs.issuer,
+    # )
```

Comment:
coming soon ... 🤔 
https://cm-jira.usa.gov/browse/LG-14277

Reply:
Yeah, that's why I left it in there